src/main/java/uk/gov/hmcts/reform/lrdapi/domain/Region.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.lrdapi.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.validation.constraints.Size;
+
+@Entity(name = "region")
+@NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+@Getter
+@Setter
+public class Region implements Serializable {
+
+    @Id
+    @Size(max = 16)
+    private String regionId;
+
+    @Column(name = "description", nullable = false)
+    @Size(max = 256)
+    private String description;
+
+    @Column(name = "welsh_description", nullable = false)
+    @Size(max = 256)
+    private String welshDescription;
```

Comment:
`description` and `welsh_description` columns are optional so they can be null.

Reply:
done